packages/ui/src/components/ColumnManager.tsx

```diff
@@ -152,9 +122,13 @@ export function ColumnManager<RowDataModel>({ table }: { table: Table<RowDataMod
         }
       }}
     >
-      <Dropdown.Toggle variant="outline-secondary" id="column-manager-button">
-        <i class="bi bi-view-list me-2" aria-hidden="true" />
-        View
+      <Dropdown.Toggle
+        variant="tanstack-table"
+        id={id}
+        // eslint-disable-next-line @eslint-react/no-forbidden-props
+        className="tanstack-table-focusable-shadow"
+      >
+        <i class="bi bi-view-list me-2" aria-hidden="true" /> View{' '}
```

Comment:
The trailing space is necessary to ensure consistent spacing between the text and the down-icon that Bootstrap renders.